docs/operations-guide.md

```diff
@@ -172,7 +172,7 @@ For the DEP "MDM server" in the environment variable $DEP_NAME (see above) this
 ##### Example usage
```

Comment:
Let's add a note like some of the other scripts that describes what the options do.

Let's also add a (very brief) note in the API endpoint in the ops guide, too.